samples/cellular/slm_shell/boards/nrf52840dk_nrf52840.overlay

```diff
@@ -28,14 +36,18 @@
 			bias-pull-up;
 		};
 		group2 {
-			psels = <NRF_PSEL(UART_TX, 1, 2)>;
+			psels = <NRF_PSEL(UART_TX, 1, 2)>,
```

Comment:
I think it would be nice to align these with the board configurations, where `group1` defines TX and RTS pins and `group2` RX and CTS pins. Also, CTS is usually configured with a pull-up? For example: https://github.com/nrfconnect/sdk-zephyr/blob/59d05a489d4fcb0b1220f74de4b94c0533e0a24e/boards/nordic/nrf9151dk/nrf9151dk_nrf9151_common-pinctrl.dtsi#L7

This applies to the other overlays as well.